data/serializer/src/test/java/tech/pegasys/teku/api/schema/BeaconValidatorsTest.java

```diff
@@ -21,20 +21,26 @@
 import org.junit.jupiter.api.Test;
 import tech.pegasys.teku.datastructures.state.BeaconState;
 import tech.pegasys.teku.datastructures.state.Validator;
-import tech.pegasys.teku.datastructures.util.BeaconStateUtil;
 import tech.pegasys.teku.datastructures.util.DataStructureUtil;
 import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+import tech.pegasys.teku.spec.SpecProvider;
+import tech.pegasys.teku.spec.StubSpecProvider;
+import tech.pegasys.teku.spec.util.BeaconStateUtil;
 import tech.pegasys.teku.ssz.SSZTypes.SSZList;
 import tech.pegasys.teku.util.config.Constants;
 
 class BeaconValidatorsTest {
   private final DataStructureUtil dataStructureUtil = new DataStructureUtil();
+  private final BeaconStateUtil beaconStateUtil =
+      StubSpecProvider.create().getGenesisSpec().getBeaconStateUtil();
+  private final SpecProvider specProvider = StubSpecProvider.create();
```

Comment:
We should just create the `SpecProvider` once:
```suggestion
  private final SpecProvider specProvider = StubSpecProvider.create();
  private final BeaconStateUtil beaconStateUtil =
      specProvider.getGenesisSpec().getBeaconStateUtil();
```

Reply:
hrm i wonder how i did that, the whole point was to get the beaconStateUtil from provider :/ good catch